definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent-prod.json

```diff
@@ -110,5 +110,68 @@
     "CaseEventID": "reissueFromIssued",
     "UserRole": "caseworker-divorce-solicitor",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "02/02/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "uploadDocument",
```

Comment:
why?
1: CRU - no updates.
<img width="670" alt="Screenshot 2021-02-02 at 16 49 14" src="https://user-images.githubusercontent.com/5647337/106625089-9e38c900-6576-11eb-8bdf-8454432279f5.png">

2: I have talked to the PET team. They need to have it sooner than later.

They will keep working on it.

Reply:
This was moved into the prod file to show no change. If the PET team need this work, they can continue to impliment it however they see fit. But it wont be a part of this story. @yarpo